spec/features/user_views_past_orders_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe "User can view past orders" do
+  let(:user) { User.create(username: "adam", password: "adam") }
+
+  it "user sees a profile page" do
+    visit '/login'
+    fill_in "Username", with: user.username
+    fill_in "Password", with: user.password
+    click_button "Login"
+
+    expect(current_path).to eq('/adam')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.